Lib/pdb.py

```diff
@@ -142,8 +142,10 @@ def check(self):
             print('Error:', self.orig, 'is a directory')
             sys.exit(1)
 
-        # Replace pdb's dir with script's dir in front of module search path.
-        sys.path[0] = os.path.dirname(self)
+        # Replace pdb's dir with script's dir in front of module search path
+        # if safe_path is not set, otherwise sys.path[0] is not pdb's dir
```

Comment:
```suggestion
        # if safe_path is not set
```

the part I removed seems a bit grammatically incorrect (and not really necessary) unless I'm missing something

Reply:
I was trying to comment the reason why we should not replace `sys.path[0]` - it was not obvious to me. We can either delete the comment or I can try to rephrase the whole thing to make more sense.

Reply:
ok, rephrase it then. I think it's too terse and not clear what that last part is talking about.

Reply:
Is it better now?